### DISCUSSION TITLE
WebXR check failing

### DISCUSSION THREAD

**Discussion by felipe-chamas** [google/model-viewer#2246] [Q&A]
### Description

A collaborator send me an issue when she tries to open WebXR. The button shows up but the AR never opens (her device does not support AR). Here's a couple debugging prints she sent me:
![image](https://user-images.githubusercontent.com/49691924/112007968-dc2c8500-8b03-11eb-882e-86ff8591a92b.png)
![image](https://user-images.githubusercontent.com/49691924/112008239-1138d780-8b04-11eb-8792-1e0662adf870.png)

It seems to be a `false positive` case...
I'm sorry for not having more information to share, but if there's any other test we can ask her to perform on devTools, just let me know! 

### Version

- model-viewer: latest - website

### Browser Affected

- [x] Chrome, version: 83.0.4103.116
- [ ] Edge
- [ ] Firefox
- [ ] IE
- [ ] Safari

### OS

- [x] Android 10 - Redmi 9
- [ ] iOS
- [ ] Linux
- [ ] MacOS
- [ ] Windows

### AR

- [x] WebXR
- [ ] SceneViewer
- [ ] QuickLook

**Comment by elalish** [comment]
Unfortunately this is a shortcoming of the WebXR API, partially driven by privacy concerns. False positives exist, and this is the best we can currently do to handle them. There is talk in Chrome of doing some work to reduce these within the API implementation. @klausw: another user would like accurate session support info.
